pagopa/pn-frontend
# PN-11636 - Handle multi language on PA

## Short description
Handle multi language on PA

## List of changes proposed in this pull request
- Add `i18next-browser-languagedetector` and create a custom detector for hash parameters

## How to test
On PA, use a direct access link and append a lang query parameter with a language (e.g. &lang=en) to it. Verify that the page is translated into the selected language. Also, try refreshing the page and check that the selected language is retained
